### ISSUE TITLE
Google Drive links do not work

### ISSUE THREAD

**Issue by Zero3K** [erickutcher/httpdownloader#72]
When I try to download one, it gives me a File IO error.

**Comment by erickutcher** [comment]
What's the full path? Do you have permission to write to that folder? You might try running the program as Administrator.

**Comment by Zero3K** [comment]
1. C:\Users\Bryan\Downloads.
2. Yes.
3. All programs are being ran as Administrator because of a registry change that I did.

**Comment by erickutcher** [comment]
That's odd. The default download directory should be that Downloads folder.

Does any other path work? Like the Desktop?

**Comment by Zero3K** [comment]
Yes, and the default download directory is C:\Users\ProfileName\Documents\Downloads.

**Comment by Zero3K** [comment]
Oh, it turned out that the file already existed in the directory that I set. It didn't ask to overwrite it for some reason.

**Comment by Zero3K** [comment]
Is that an actual issue or do I have it configured wrongly?

**Comment by erickutcher** [comment]
Do you have the program set to use a temporary download directory? If that's set and it attempts to move the file from the temp directory into the selected download directory and a file of the same name already exists, then the download will be set to Stopped. If there's a File IO Error, then it probably means there's a permission issue or that the file is open somewhere else.

**Comment by Zero3K** [comment]
1. No, I don't have it set to use that option.
2. The file wasn't open in any other application.

**Comment by erickutcher** [comment]
Antivirus software maybe? I'm not sure. I'm not able to recreate the issue. Try doing a similar setup on another computer.

**Comment by Zero3K** [comment]
I don't have any other computer to use. Maybe it was open in another program and I just don't remember. If it happens again, I'll let you know.

**Comment by Zero3K** [comment]
I did notice the error occur when I did the following:

1. Download and install https://github.com/maharmstone/btrfs.
2. Make a 512 MB VHD and attach it.
3. Format the VHD as FAT.
4. Open a Command Prompt window.
5. Run format vhd_drive (the drive that the VHD is attached to) /fs:BTRFS
6. Try to download a file to the drive.
